packages/error/src/index.ts

```diff
@@ -0,0 +1,21 @@
+const isProduction: boolean = process.env.NODE_ENV === "production";
+const prefix = "\nVisit https://community.frontity.org for help! 🙂\n";
+
+// Throw an error if the condition fails
+// Strip out error messages for production
+// > Not providing an inline default argument for message as the result is smaller
+export default function invariant(condition: boolean, message?: string) {
+  if (condition) {
+    return;
+  }
+  // Condition not passed
+
+  if (isProduction) {
+    // In production we strip the message but still throw
+    throw new Error(prefix);
```

Comment:
Do we want to throw the "visit community" message in production? (if so, I guess the babel plugin won't help).

Reply:
No, I think not! I think we don't show the warning, and if it's an error then we just throw the error and only in __DEV__ we have a full error message.

Reply:
Actually, I think we can throw it. I guess that's as good as throwing any other message.